schemas/collections.ts

```diff
@@ -0,0 +1,87 @@
+import { z } from 'zod'
+
+export const TimestampSchema = z.object({
+  _seconds: z.number(),
+  _nanoseconds: z.number(),
+})
+
+/** Schemas from DB
```

Comment:
Could you, please add this as a TODO and create a ticket to update these schemas?

Reply:
https://linear.app/brinca/issue/BRINCA-83/schemas-from-db